src/getExtension/getExtension.ts

```diff
@@ -24,3 +24,24 @@ export function getExtensions(): Promise<
 
   return documentReadyPromise(() => apiWindow.kilt)
 }
+
+/**
+ * This function enables the communication with extensions supporting the Credential API.
+ *
+ * The `meta` property of `window.kilt` is set according to the Credential API.
+ * After this is done an event is dispatched to notify all extensions that they should inject themselves now.
+ */
+export function initializeKiltExtensionAPI() {
+  apiWindow.kilt = apiWindow.kilt || {}
+
+  Object.defineProperty(apiWindow.kilt, 'meta', {
+    value: {
+      versions: {
+        credentials: '3.0'
```

Comment:
Should the specific version of the API be customizable? We also have 3.1 now and 3.2 is in the works.

Reply:
I think we would always update this library with the new types/changed API. So if you want to support 3.1 or 3.2 you would update this library and get the changes as part of the library.